commodore/helpers.py

```diff
@@ -184,6 +184,10 @@ def relsymlink(src: P, dest_dir: P, dest_name: Optional[str] = None):
     # https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.relative_to
     link_src = os.path.relpath(src, start=dest_dir)
     link_dst = dest_dir / dest_name
+    if not P(link_src).exists():
```

Comment:
Isn't `link_src` already a Path?
```suggestion
    if not link_src.exists():
```

Reply:
`relpath` returns type `Text`.